### ISSUE TITLE
result type Double can't be cast to the desired output type Long

### ISSUE THREAD

**Issue by Ryuk17** [pytorch/pytorch#38916]
I meet this error when running the following code, however, the type of loss is torch.float64. I don't know what induces this error,  can anyone tell me why?
```
            loss = criterion(label, target)
            loss.backward()
```

**Comment by ezyang** [comment]
Can you post a more complete script that reproduces your error?

**Comment by ezyang** [comment]
I'm going to close this as not actionable as it stands, feel free to reopen when you can provide more information.
